mobile/pubspec.lock

```diff
@@ -1022,5 +1022,5 @@ packages:
     source: hosted
     version: "2.0.3"
 sdks:
-  dart: ">=2.19.0 <4.0.0"
+  dart: ">=2.19.0 <3.0.0"
```

Comment:
🛠️ I don't think this should be in here. I keep running into this problem as well on my machine (somehow it is always changed back; but I think for @bonomat it is then changed back to 4.0.0 😅)

Reply:
I can remove it, but according to the `pubspec.yaml`, this change is correct:

```yaml
environment:
  sdk: ">=2.17.5 <3.0.0"
```

If you play around with those constraints and you run `flutter pub get` the `pubspec.lock` changes as expected.